packages/widget-chat/src/modules/redux-spark/spark.js

```diff
@@ -19,3 +19,7 @@ export default function createSpark(accessToken) {
     }
   });
 }
+
+const spark = createSpark();
```

Comment:
Yup. That way we can just grab the same `spark` object by importing spark. You can still request the `createSpark` method, like in tests when you may actually want to create a new spark object every time.